File: Tugas coding modul 3/coding python/PRAK304-2310817120005-Noviana_Nur_Aisyah.py
q = int(input('Input \n'))

if q == 0:
    print('\nOutput \nNol')
elif q > 0 and q <= 10:
    print('\nOutput \nSatuan')
elif q >= 11 and q < 20:
    print('\nOutput \nBelasan')
elif q >= 20 and q <= 99:
    print('\nOutput \nPuluhan')
else:
    print('\nOutput \nAnda Menginput Melebihi Limit Bilangan')

# 3 0 100 62 13

File: Tugas coding modul 3/coding c/PRAK304-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

void main(){
    int b;

    printf("Input\n");
    scanf("%d", &b);

    if (b == 0){
        printf("\nOutput \nNol");
    } else if (b > 0 && b < 10){
        printf("\nOutput \nSatuan");
    } else if (b >= 10 && b < 20){
        printf("Output \nBelasan");
    } else if (b >= 20 && b < 100){
        printf("\nOutput \nPuluhan");
    } else {
        printf("\nOutput \nAnda Menginput Melebihi Limit Bilangan");
    }
}

// 3 | 0 | 100 | 62 | 13